Orange/widgets/unsupervised/owdistancefile.py

```diff
@@ -158,7 +160,12 @@ def parametersFromFile(self, path):
         return {"recent_paths": stored_recent_paths_prepend(self.WIDGET, r)}
 
     def canDropFile(self, path: str) -> bool:
-        return os.path.splitext(path)[1].lower() in (".dst", ".xlsx")
+        try:
+            distances = DistMatrix.from_file(path)
+        except Exception:  # pylint: disable=broad-except
+            return False
+        else:
+            return distances.shape[0] == distances.shape[1]
```

Comment:
Have we agreed on that?

Reply:
Yes. The widget also tests this (and shows an error) when loading the file.